### PR TITLE
GraphQL

### PR DESCRIPTION
**Pull request by patrickocoffeyo**
This is FAAAR from done, just putting this PR up so @chasingmaxwell can see the small amount of "progress" I made

### CODE REVIEW

File: src/graphql/typeDefs/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { readFile } from 'fs-extra';
```
Comment (patrickocoffeyo) [inline src/graphql/typeDefs/index.ts]: I was in the middle of replacing this with `graphql-import`'s `importSchema` tool, so this is going to be deleted soon.

File: src/handlers/express.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { ApolloServer } from 'apollo-server-express';
```
Comment (chasingmaxwell) [inline src/handlers/express.ts]: putting this in a `handlers` directory feels kinda like a lambda. Maybe this should just be in `src/server.ts` or something?

File: src/handlers/util/getExecutableSchema.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { GraphQLSchema } from 'graphql';
```
Comment (chasingmaxwell) [inline src/handlers/util/getExecutableSchema.ts]: I could see this file moved to `src/graphql/util/`. What do you think?

**Comment by patrickocoffeyo** [comment]
Everything looks good! :shipit:

**Pull request merged**